django_ninja_crudl/endpoints/delete.py

```
@@ -0,0 +1,78 @@
+"""CRUDL API base class."""
```

Ruff (0.8.2)

File `django_ninja_crudl/endpoints/delete.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)